pkg/converter/profileConverter.go

```diff
@@ -0,0 +1,33 @@
+package converter
+
+import (
+	"github.com/projectcalico/libcalico-go/lib/api/unversioned"
+	"github.com/projectcalico/libcalico-go/lib/api"
+	"k8s.io/client-go/pkg/api/v1"
+
+)
+const NS_POLICY_ANNOTATION = "net.beta.kubernetes.io/network-policy"
+// Format to use for namespace profile names.
+const NS_PROFILE_FMT = "k8s_ns.%s"
+type profileConverter struct{
+
+
+}
+
+func (p *profileConverter)convert(k8sObj interface{})(interface{},error){
+
+	// Converting obj to calico representation
+	labels := k8sObj.(*v1.Namespace).Labels
+	profile := api.Profile{
```

Comment:
You probably want to call `NewProfile` here instead: https://github.com/projectcalico/libcalico-go/blob/master/lib/api/profile.go#L62